__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const app = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+const request = require('supertest')
+
+
+afterAll(() => {
+    db.end()
```

Comment:
You may still be getting the yellow messaging in the terminal as this is missing a return?